docs/home_page_example.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
```

Comment:
I would put this elsewhere, the `docs` folder will be versioned at some point (it will be copied to https://github.com/apify/crawlee/tree/master/website/versioned_docs for each feature release), but the homepage won't. I actually wanted to look at this rather sooner and at least have "latest stable and latest dev" so we don't confuse users with unreleased things in docs.

Reply:
I am wondering where to place it. The `website/` dir is currently excluded from the Python static analysis. And I am not sure whether we want to include it because of `node_modules`. Any suggestions?

Reply:
You could include just `*.py` files maybe? Where is that exclusion defined?

Reply:
In the `pyproject.toml` and `makefile`. 

I moved it to the `website/`, tested static analysis then, and it seems that there is no noticeable slowdown.